frontend/features/Facility/List/Review/index.tsx

```diff
@@ -0,0 +1,34 @@
+import { FacilityWithRelations } from '@/types/generated/api';
+import { Star } from '../../Star';
+import { truncate } from '@/utils/truncate';
+
+type FacilityProps = {
+  facility: FacilityWithRelations;
+};
+
+export const FacilityReviewList = ({ facility }: FacilityProps) => {
+  return (
+    <div className="px-4">
+      <h3 className="text-lg font-semibold text-gray-700 mb-2">💬口コミ</h3>
+      {facility.reviews && facility.reviews?.length > 0 ? (
+        facility.reviews?.map(review => (
```

Comment:
細かいけどfacility.reviews && って条件分岐しているから?は不要なはず

```suggestion
        facility.reviews.map(review => (
```

Reply:
13行目もかね？

f1081d37672d6ddc019670b67ca355110e71ee40